CONTRIBUTING.md

```diff
@@ -16,6 +16,8 @@ Remember that you can continue to push to your `feature/*` or `fix/*` branches t
 
 Please note that your pull request will have to pass the CI tests in order to be merged.
 
+Including unit tests is very welcomed and highly recommended. xpcc has its own simple and easy to use unit test framework. Unit tests are placed in the `test` subdirectory of every component (e.g. `xpcc/src/xpcc/math/filter/test`) and are run with `scons unittest`. See TESTING.md for more information about testing strategies in xpcc.
+
```

Comment:
Also, in that text up there, link to the files (ie. TESTING.md and source folder), so that people don't have to search for it.
If they looking at this file, they will probably already be on the website.